### DISCUSSION TITLE
The Spread Operator ||  Spread Syntax

### DISCUSSION THREAD

**Discussion by fimoefive** [nss-evening-cohort-14/evening-client-side#93] [Look at what I learned!]
I know several of you have referred to this(...) as the "ELIPSIS" which is the English language definition, but not within the JavaScript language.
Ellipsis: 
"An ellipsis consists of either three or four periods, or dots. A single dot is called an ellipsis point. An ellipsis can indicate the omission of words in the middle of a quoted sentence or the omission of sentences within a quoted paragraph. ..."

JavaScript: Spread Operator
[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax](url)

**Comment by drteresavasquez** [comment]
Well, hello! 😄 

Check this out as well: https://github.com/nss-evening-cohort-14/evening-client-side/discussions/86
